services/cloudca/network_acl_rule.go

```diff
@@ -0,0 +1,110 @@
+package cloudca
+
+import (
+	"encoding/json"
+
+	"github.com/cloud-ca/go-cloudca/api"
+	"github.com/cloud-ca/go-cloudca/services"
+)
+
+type NetworkAclRule struct {
+	Id           string `json:"id,omitempty"`
+	NetworkAclId string `json:"networkAclId,omitempty"`
+	RuleNumber   string `json:"ruleNumber,omitempty"`
+	Cidr         string `json:"cidr,omitempty"`
+	Action       string `json:"action,omitempty"`
+	Protocol     string `json:"protocol,omitempty"`
+	StartPort    string `json:"startPort,omitempty"`
+	EndPort      string `json:"endPort,omitempty"`
+	IcmpType     string `json:"icmpType,omitempty"`
+	IcmpCode     string `json:"icmpCode,omitempty"`
+	TrafficType  string `json:"trafficType,omitempty"`
+	State        string `json:"state,omitempty"`
```

Comment:
ACS returns one (Active or Inactive) and CloudMC parses it. That's why it's also here, although we don't use it.